test/functional/crud_api_tests.js

```diff
@@ -1023,6 +1023,44 @@ describe('CRUD API', function() {
     }
   });
 
+  it('should correctly throw error if update doc for findOneAndUpdate lacks atomic operator', {
+    // Add a tag that our runner can trigger on
+    // in this case we are setting that node needs to be higher than 0.10.X to run
+    metadata: {
+      requires: { topology: [] }
```

Comment:
FYI, if you omit this section completely it will just run on all topologies. Also, when you do it this way you can revert to OG mocha-style:
```
it('should correctly throw error if update doc for findOneAndUpdate lacks atomic operator', function(done) {
  // test
});
```